modules/virt-creating-linux-bridge-nad-cli.adoc

```diff
@@ -49,7 +50,8 @@ spec:
 <5> The name of the Linux bridge configured on the node.
 <6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
 <7> Optional: The VLAN tag. No additional VLAN configuration is required on the node network configuration policy.
-<8> Optional: Indicates whether the VM connects to the bridge through the default VLAN. The default value is `true`.
+<8> Optional: Specifies the state of the container interface (virtual ethernet device peer inside the network namespace). When set to `true`, the container interface is down. The default value is `false`. The `disableContainerInterface` field is relevant for IPv6 single-stack and IPv4/IPv6 dual-stack environments. You cannot use IP Address Management (IPAM) when this field is set to `false`.
```

Comment:
I am not sure what _virtual ethernet device peer inside the network namespace_ means, but I am not a typical user. Could this phrase be worded differently, or is it OK for our users?

Reply:
Whatever you decide, I have had other peer reviewers ask me to remove the parenthesis when possible (I am not sure why.)

Is this right?
```suggestion
<8> Optional: Specifies the state of the container interface, which is a virtual ethernet device peer inside the network namespace. When set to `true`, the container interface is down. The default value is `false`. The `disableContainerInterface` field is relevant for IPv6 single-stack and IPv4/IPv6 dual-stack environments. You cannot use IP Address Management (IPAM) when this field is set to `false`.
```

Reply:
@ormergi Does the above suggested wording change make sense? I wanted to confirm before updating the PR. Thanks!